buildSrc/src/main/kotlin/Testing.kt

```diff
@@ -106,11 +106,10 @@ fun Project.nessieConfigureTestTasks() {
           )
         } else {
           // Manage to specific Nessie version
-          val newGroupIds = nessieServerVersionToUse.isHigherVersionThan("0.50")
```

Comment:
We no longer test Nessie < 0.50, so the code can go away (also because that `VersionNumber` class is deprecated for removal w/ Gradle 9).